src/xtgeo/xyz/points.py

```diff
@@ -138,22 +138,6 @@ def __lt__(self, value):
     def __le__(self, value):
         return self.dataframe[self.zname] <= value
 
-    # ----------------------------------------------------------------------------------
-    # Properties
-    # ----------------------------------------------------------------------------------
-
-    @property
-    def pname(self):
-        """ Return or set the name of the POLY_ID column"""
-        return self._pname
-
-    @pname.setter
-    def pname(self, pname):
```

Comment:
Moved to `XYZ` instead, we now also verify that the name reflects a column in the df (like the other properties does)